index.html

```diff
@@ -222,15 +222,16 @@ <h2>Contact Me</h2>
                   type="text"
                   name="name"
                   id="full-name"
-                  required=""
+                  required
```

Comment:
- [ ] [ OPTIONAL] it would be better if you use the right font-size for the all input in order to match the Figma design

Reply:
@youmari thanks, noted!